### PR TITLE
RSDK-899 import proto methods from typescript-sdk

### PR DESCRIPTION
**Pull request by maximpertsov**
JIRA: https://viam.atlassian.net/browse/RSDK-899

Stop generating JS protobufs and instead import them from our typescript sdk.

Questions:
* [x] Can we stop tracking assets in `web/runtime-shared` now?
  * Not yet unfortunately - see https://viam.atlassian.net/browse/RSDK-900 for details

TODOs
* [x] Depends on: https://github.com/viamrobotics/viam-typescript-sdk/pull/2

### CODE REVIEW

**Review by micheal-parks — APPROVED** [review]
😍

File: Makefile [hunk 1]
```diff
@@ -41,16 +42,6 @@ tool-install:
 		gotest.tools/gotestsum \
 		github.com/rhysd/actionlint/cmd/actionlint
 
-buf: buf-web
-
-buf-web: tool-install
```
Comment (edaniels) [inline Makefile:46]: reminder to self, will need to add sessions proto to sdk instead of using buf here

**Review by edaniels — APPROVED** [review]
LGTM

File: Makefile [hunk 2]
```diff
@@ -21,7 +20,9 @@ build: build-web build-go
 build-go:
 	go build $(GO_BUILD_TAGS) ./...
 
-build-web: buf-web
+build-web:
+	npm ci --audit=false
```
Comment (DTCurrie) [inline Makefile:24]: Do we still need to install/maintain any `npm` packages at the root level of the project? That single dependency was for how we previously generated `buf` stuff.

I imagine we can delete the `package.json` file at the root and just keep the one in `web/frontend`.

**Review by DTCurrie — APPROVED** [review]
✅ just a question about dependencies

File: Makefile (hunk 2 above)
Comment (maximpertsov) [reply Makefile:24]: great call -- yes the top-level dependency is only for generating protobufs, which this PR removes

**Pull request merged**